### PR TITLE
Jie's update

### PR DESCRIPTION
**Pull request by jimmyyou**

### CODE REVIEW

**Review by Romero027 — COMMENTED** [review]
Can you add a pdf of Terra? :)
Also, please run `hexo server` to check if everything is correct. The abstract is not showing now.

**Comment by Aetf** [comment]
> Also, please run `hexo server` to check if everything is correct. The abstract is not showing now.

The command is `npm run server`. `hexo server` won't work if you haven't installed it globally

File: source/_data/MCPubs.bib [hunk 1]
```diff
@@ -303,6 +303,9 @@ @Article{terra:arxiv19
...
+  },
```
Comment (Aetf) [inline source/_data/MCPubs.bib:308]: No comma at the end. I think that's why the parsing is broken.

**Review by Aetf — CHANGES_REQUESTED** [review]

**Comment by jimmyyou** [comment]
remove pull request, re-submitting a new one

**Pull request closed without merging**